drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-client/src/test/java/org/drools/workbench/screens/testscenario/client/ScenarioEditorPresenterTest.java

```diff
@@ -167,8 +168,15 @@ public void testRunScenario() throws Exception {
 
         presenter.onRunScenario();
 
-        verify(view).initKSessionSelector(eq(path), any(Scenario.class));
-        verify(view).showAuditView(anySet());
+        InOrder inOrder = inOrder( view );
```

Comment:
Not sure if it a good practice to test the order, but in this case when we are in a hurry I rather do that, than refactor to something where testing the order is not needed.